CHANGELOG.md

```diff
@@ -1,3 +1,34 @@
+# Next release
+
+* [CHANGE] Deprecate specification of Prometheus rules via ConfigMaps in favor of `PrometheusRule` CRDs
+
+## 0.20.0 / 2018-06-05
+
+With this release we introduce a new Custom Resource Definition - the
+`PrometheusRule` CRD. It addresses the need for rule syntax validation and rule
+selection accross namespaces. `PrometheusRule` replaces the configuration of
+Prometheus rules via K8s ConfigMaps. There are two migration paths:
+
+1. Automated live migration: If the Prometheus Operator finds Kubernetes
+   ConfigMaps that match the `RuleSelector` in a `Prometheus` specification, it
+   will convert them to matching `PrometheusRule` resources.
+
+2. Manual migration: We provide a basic CLI tool to convert Kubernetes
+   ConfigMaps to `PrometheusRule` resources.
+
+```bash
+go get -u github.com/coreos/prometheus-operator/cmd/po-rule-cm-to-rule-file-crds
```

Comment:
should we be renaming this? `po-rule-cm-to-prometheus-rule-crds`?

Reply:
po-rules-migration

Reply:
Works for me. @brancz would you mind addressing this in https://github.com/coreos/prometheus-operator/pull/1430 ?